client/src/App.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import SearchBar from './components/SearchBar/SearchBar.jsx';
+import Button from 'react-bootstrap/Button';
 
 const App = function() {
```

Comment:
Will App ever have a state? I am not too familiar with Hooks, which you have been using, but I used the a class to declare app and then extended React.Component.

Reply:
App probably will need state eventually, yes. It might work best to keep the fetched results of API calls in state in App.jsx and then pass state to QuestionList as a prop and pass function(s) for updating state to SearchBar. I was planning on using hooks in App when I get to that point, which means I can just import the useState hook and call it to add state.